### ISSUE TITLE
Switching Material themes messes up IDE settings

### ISSUE THREAD

**Issue by LoganDark** [ChrisRM/material-theme-jetbrains#1618]
### Summary

Selecting a new theme from the dropdown in Appearance & Behavior > Material Theme messes with settings.

### Expected behavior

Syntax highlighting settings & accent color settings stay the same.

### Actual behavior

Settings are messed with in 2020.1.2, including accent color settings and syntax highlighting settings.

However, switching to a different theme and back in 2020.3.3 (plugin version 5.3.0.1) does not have this behavior.

### Steps to reproduce (if needed)

1. Back up your settings
2. Go to Appearance & Behavior > Material Theme
3. Change Selected Theme to something else

This:

![image](https://user-images.githubusercontent.com/4723091/87455858-ef717300-c5ba-11ea-9f79-a471317d8f21.png)

Becomes this:

![image](https://user-images.githubusercontent.com/4723091/87455894-fc8e6200-c5ba-11ea-913c-7208e3eef5f7.png)

Since this may be an issue with my specific settings, here is an export of all of my settings: [settings.zip](https://github.com/ChrisRM/material-theme-jetbrains/files/4920260/settings.zip)

To reproduce with my settings, try to change the theme to Material Darker.

### Environment

OS: macOS 10.14.6
Product: CLion 2020.1.2
Plugin version: 5.3.2.1
Plugin configuration: [material_theme.xml.2020.1.2.zip](https://github.com/ChrisRM/material-theme-jetbrains/files/4920243/material_theme.xml.2020.1.2.zip)

**Comment by LoganDark** [comment]
Related to #1617

**Comment by Monadic-Cat** [comment]
Are you creating multiple duplicates of the same issue?

...please don't do that.

**Comment by LoganDark** [comment]
@Monadic-Cat No. The first issue was closed because of the tone, so I created a new issue without 'the tone'. And these are separate issues, so I created 2 issues.

The first issue is for the theme changing to Darcula, the second one is for changing the theme messing up settings.

**Comment by mallowigi** [comment]
I don't really understand this issue. Are you reporting that the color scheme definitions between the 2018 version and the 2020 version are different? It could be, two years is a long time.

In any case you can still save a copy of your 2018.3 scheme and it will appear as untouched in 2020.2

**Comment by LoganDark** [comment]
No, the color scheme looks fine in 2020.1.2. But thanks to #1617, the theme is Darcula, which is not what I want. Trying to change it to Material Darker causes the color scheme to mess up as shown in the image.

Even if #1617 is a wontfix, this is definitely some kind of bug since it could bite anyone who tries to switch their theme.

**Comment by mallowigi** [comment]
Again, it's not a mess up. From what I see in the screenshot, the only difference is that keywords are italic, caret color is orange and the font is not Fira Code. And that's exactly some of the changes in the color schemes that occured during those two years.

**Comment by LoganDark** [comment]
> the font is not Fira Code

hasklig :D

> And that's exactly some of the changes in the color schemes that occured during those two years.

Well to be fair Material Darker always had an orange caret. I changed it to blue because I liked the blue better.

**Comment by mallowigi** [comment]
But this is troubling. Are you saying that you had a copy of Material Darker with those changes and that these changes did not get propagated from 2018 to 2020 ?

**Comment by LoganDark** [comment]
@mallowigi I did not copy Material Darker. I just went to change the caret color in the actual caret color settings.

**Comment by mallowigi** [comment]
Ok so everything is okay then ?

**Comment by LoganDark** [comment]
Yes, now everything is okay. As long as I don't touch anything. Ever.
